app/views/groups/manage/manage_existing_groups.scala.html

```diff
@@ -80,7 +81,7 @@
 
         @summaries.map { summary =>
 
-            @h2(msgs(summary.groupName) + " " + tag(summary.isCustomGroup).toString)
+            @h2(msgs(summary.groupName) + " " + groupTypeTag(Option(summary.taxService).isEmpty).toString)
```

Comment:
Think here - this shouldn't be wrapped in an option if summary.taxService is optional now?